doc/recognition.xml

```diff
@@ -416,6 +419,19 @@ elements that are generated from them remember, how they were acquired.
 </Description>
 </ManSection>
 
+<ManSection>
+<Attr Name="HomomSourceCheck" Arg="ri"/>
+<Description>
+    The value of this attribute is a function which returns a boolean
+    which is <K>false</K> for values which are not in the source of
+    <Ref Attr="Homom"/>. For example, if this node represents
+    a matrix group that preserves a subspace, then matrices which do not
+    preserve that subspace may cause <Ref Attr="Homom"/> to produce
+    incorrect answers. If not defined, this function defaults
```

Comment:
OK, so it seems this is about incorrect answers, not about calls to `Error()`?

Reply:
incorrect answers, calls to Error, break loops,... I've seen all 3. But not sure the best way of describing that.